### ISSUE TITLE
Suggestion: add parameter to get OSM labels in different languages

### ISSUE THREAD

**Issue by iriberri** [CartoDB/CartoDB-basemaps#21]

**Comment by almccon** [comment]
Yes, I've wanted this for a long time! At Stamen we'd also like to do this for our Toner style: https://github.com/stamen/toner-carto/issues/13

**Comment by GabeMedrash** [comment]
This would be fantastic. Noticed that if I zoom in far enough in China, for example, I see labels in Chinese, but at higher zoom levels it's all English.
